README.md

```diff
@@ -5,3 +5,44 @@ This is the Git repo of the Docker [official image](https://docs.docker.com/dock
 ## Release Process
 
 The release process is documented on our [Wiki Page](https://github.com/ExpressGateway/express-gateway/wiki/Express-Gateway-Release-Process)
+
+## Using this docker image
+
+Obtain a local copy of the image:
+```
+docker pull expressgateway/express-gateway
```

Comment:
Again, this must be `docker pull express-gateway` — when the official PR will be merged.

Reply:
Not sure that I am following.  The image up there today requires that you provide the docker repository - e.g. expressgateway/<image name>.  A pull of an "official" image would work after you have an "offical" repository.  But, all I see is a public repo that you have created, so still needs the repo name in the pull.

[@localhost docker-express-gateway]$ docker images | grep express-gateway
**expressgateway**/express-gateway                                   latest              41604254e8c9        12 days ago         185MB
[j@localhost docker-express-gateway]$ docker pull express-gateway
Using default tag: latest
Error response from daemon: pull access denied for express-gateway, repository does not exist or may require 'docker login'
[j@localhost docker-express-gateway]$ 

Is there an official docker hub repo that I am not seeing?  Thanks.

Reply:
No — we do not have an official docker image yet, and that's the whole point. This repository is meant to store the official docker file definition, and it should not be used by anybody yet.

Does that clarify?

Reply:
That explains why you are suggesting that change.  However, it is out there and public.  So, if you tell people to pull without the repo name it will fail.  Can I suggest that you make the current repo private until you are ready to make official ... or I will make the change you want and add a note explaining it ... somewhere ... that can be removed when the repo is granted official status.  I would be concerned with publishing instructions in a public hub repo that will not currently work.   Thoughts?

Reply:
> So, if you tell people to pull without the repo name it will fail. 

I'm not suggesting that — the current `README` it's not stating any installation process at all (because we're missing the last step)

> Can I suggest that you make the current repo private until you are ready to make official

Unfortunately I can't — it is required to leave it public so the automatic scripts on `docker-library` can be triggered for verification.

My plan was to approve the PR and defer the merge when this will be finally finalized. Does that sound good?

Reply:
Sounds good.  Making the change.  Thanks for the clarity on how you are planning to roll this out.